deploy.yml

```diff
@@ -3,10 +3,10 @@
 # Provision Ansible controller with SSH key for remote
 - hosts: localhost
   roles:
-    - ssh-key
-  vars:
-    ssh_key_name: ansible_remote
-    hashicorp_vault_private_key_path: "/applications/heritage-{{ environment_name }}-eu-west-2/fil-tuxedo/aws"
+    - role: ssh-key
+      vars:
+        ssh_key_name: ansible_remote
+        hashicorp_vault_private_key_path: "/applications/heritage-{{ environment_name }}-eu-west-2/fil-tuxedo/aws"
```

Comment:
`vault_base_path`?

Reply:
Fixed in a679eb3412b9ea77dad865779c0d693dda2135b7.